scantool/diag.h

```diff
@@ -138,7 +138,7 @@ typedef uint16_t flag_type;	//this is used for L2 type flags (see diag_l2.h)
  * Only applicable if DIAG_L1_DOESKEEPALIVE is set
  *
  * data = (struct diag_msg *).
- * The (struct diag_msg)->data member must be a raw message (including headers, checksums etc)
+ * The (struct diag_msg)->data member must be a raw message (including headers)
  *
```

Comment:
I think the raw message should include the checksum anyway, even if l0_elm doesn't need it.